mkdocs.yml

```
@@ -0,0 +1,45 @@
+            filters:
+              - "!^_"
+              - "!CountryHoliday"
+  - gen-files:
+      scripts:
+        - scripts/gen_ref_pages.py
+        - scripts/summary.py
+  - literate-nav:
+      nav_file: SUMMARY.md
+
+markdown_extensions:
+    - markdown_include.include:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 4

(indentation)